src/styles/blocks/catalog.scss

```diff
@@ -12,14 +12,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform 0.3s, color 0.3s;
```

Comment:
We can use a variable for the duration of the transform because all our elements on the page change with the same duration. But it's just a recommendation:)